sql-plugin/src/main/330+/scala/com/nvidia/spark/rapids/shims/GpuTypeShims.scala

```diff
@@ -160,4 +166,15 @@ object GpuTypeShims {
     }
   }
 
+  def isDayTimeInterval(dt: DataType) : Boolean = dt match {
+    case _: DayTimeIntervalType => true
+    case _ => false
+  }
+
+  def isYearMonthInterval(dt: DataType) : Boolean = dt match {
```

Comment:
```suggestion
  def isYearMonthIntervalType(dt: DataType) : Boolean = dt match {
```
This can be simplified as
```
def isYearMonthInterval(dt: DataType) : Boolean = dt.isInstanceOf[YearMonthIntervalType]
```

Reply:
done